pkg/gce-pd-csi-driver/controller.go

```diff
@@ -298,7 +305,7 @@ func (gceCS *GCEControllerServer) ControllerPublishVolume(ctx context.Context, r
 		PublishContext: nil,
 	}
 
-	_, err = gceCS.CloudProvider.GetDisk(ctx, volKey)
+	_, err = gceCS.CloudProvider.GetDisk(ctx, volKey, gce.V1)
```

Comment:
why is this definitely `V1`? Couldn't it be a alpha multiwriter disk that we want to get here

Reply:
The v1 API returns disks that use alpha features, it just doesn't have any information with respect to those features. Here the API call is used to detect the existence of the disk. The disk itself is thrown away and only the error code is parsed. I didn't see a need to use anything other than v1.